sky/shell/gpu/direct/rasterizer_direct.cc

```diff
@@ -52,12 +52,19 @@ void RasterizerDirect::Setup(PlatformView* platform_view,
   // The context needs to be made current before the GrGL interface can be
   // setup.
   bool success = platform_view->ContextMakeCurrent();
+  if (success) {
+    success = ganesh_canvas_.SetupGrGLInterface();
+    if (!success)
+      LOG(ERROR) << "Could not create the GL interface";
```

Comment:
I'd like to figure out how to log a bunch of GPU information or other understanding of why we couldn't create the context.  Presumably Skia folks can tell us how.  I'll file a follow-up bug.